src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+
+  color: $main-text-color;
+
+  background: $white-background;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    height: 134px;
+    width: 160px;
+    margin: 0 3px 40px;
+  }
+
+  &__product {
+    margin-bottom: 4px;
+
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
```

Comment:
I'd create a variable here and insert it into the rows 31-33 aswell as 46-48

Reply:
I don't think this is neccesary because it's two diffirent elements that not connected and those values can be changed in future.